.aqua/aqua.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+# checksum:
+#   enabled: true
+#   require_checksum: true
+#   supported_envs:
+#   - all
+registries:
+- type: standard
+  ref: v4.41.1 # renovate: depName=aquaproj/aqua-registry
```

Comment:
Since renovate is not enabled in this repo is there an alternative or do I just need to update it myself every so often?

Reply:
You could look at only use renovate for this one feature if that's all you want. I'd be glad to do a renovate config for you that's just aqua self updating the registry if you like, else I have a zsh alias that bumps this manually for me when I'm wanting it if that would help.

Reply:
Either of those would work yea.

Reply:
aqua now has `aqua update` so periodically as desired you can just update your file in the repo with a command. Renovate is still a great option, but this shouldn't block you from using now.